src/Server/PostgreSQLHandler.cpp

```diff
@@ -540,7 +540,8 @@ bool PostgreSQLHandler::processCopyQuery(const String & query)
             format_ptr->write(materializeBlock(block));
             format_ptr->flush();
             output_buffer.finalize();
-            message_transport->send(PostgreSQLProtocol::Messaging::CopyOutData(std::move(result_buf)));
+            message_transport->send(PostgreSQLProtocol::Messaging::CopyOutData(result_buf));
```

Comment:
Maybe it's better to move it and then explicitly initialize it after.
Extra empty vector allocation looks better than a full buffer copy before send